PowerShell.Common.props

```diff
@@ -111,6 +111,27 @@
     <HighEntropyVA>true</HighEntropyVA>
   </PropertyGroup>
 
+  <PropertyGroup>
+    <StronglyTypedOutputFolder>gen</StronglyTypedOutputFolder>
+  </PropertyGroup>
+
+  <Target Name = "CreateDirectories" BeforeTargets = "BeforeResGen">
+    <MakeDir
+       Directories = "$(StronglyTypedOutputFolder)"/>
+  </Target>
+
+  <ItemGroup>
+    <!-- We skip this for Microsoft.Management.UI.Internal because we have to use a workaround for WPF project. -->
+    <EmbeddedResource Update = "resources/*.resx" Condition = "'$(MSBuildProjectName)' != 'Microsoft.Management.UI.Internal'">
```

Comment:
comment that this is a workaround for https://github.com/microsoft/msbuild/issues/4751

Reply:
Done.